src/main/java/edu/harvard/iq/dataverse/dataaccess/GlobusAccessibleStore.java

```diff
@@ -0,0 +1,68 @@
+package edu.harvard.iq.dataverse.dataaccess;
+
+import jakarta.json.Json;
+import jakarta.json.JsonArray;
+import jakarta.json.JsonArrayBuilder;
+
+public interface GlobusAccessibleStore {

```

Comment:
This ended up being all static methods related to the Globus store. It could be a Util class instead?